.ci/install_kata_image_nosystemd.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
```

Comment:
You may need to edit/update the copyright line

Reply:
you mean only upgrade 2019 to 2020? @grahamwhaley

Reply:
yes, at minimum update the date.
Also, you can add your own company/name if you have made substantial edits - if you want to :-)

Reply:
thanks @grahamwhaley